.swiftlint.yml

```diff
@@ -23,8 +23,9 @@ opt_in_rules: # some rules are only opt-in
 excluded:
   - Carthage
   - Pods
-  - VaporDemo/.build
-  - PerfectDemo/.build
+  - Demos/VaporDemo
+  - Demos/PerfectDemo
+  - Tests
```

Comment:
Why do we exclude these folders?

Reply:
We don't need swiftlint in the demos and tests folder at the moment. I can open a new PR to sort them all later and reintroduce them back to the swiftlint.

Reply:
Hey, really sorry @ppamorim but linting is bound to a project as a whole not only the source code. It creates consistency and makes contributing easier. 
It would be weird if I don't have to follow the same rules in the tests folder than in the source folder.